datadog/agent/install.go

```diff
@@ -68,11 +67,12 @@ func NewInstaller(vm vm.VM, options ...func(*Params) error) (*Installer, error)
 	serviceManager := os.GetServiceManager()
 	for _, cmd := range serviceManager.RestartAgentCmd() {
 		restartAgentRes := commonNamer.ResourceName("restart-agent")
+		cmdPulumiStr := pulumi.String(cmd)
 		lastCommand, err = runner.Command(
 			restartAgentRes,
 			&command.Args{
-				Create:   pulumi.String(cmd),
-				Triggers: pulumi.Array{pulumi.String(utils.StrHash(cmd, configHash, integsHash))},
+				Create:   cmdPulumiStr,
+				Triggers: pulumi.Array{utils.PulumiStrHash(cmdPulumiStr, configHash, pulumi.String(integsHash))},
```

Comment:
Can you explain why you need `PulumiStrHash`. Can't `pulumi.Array` contains directly `cmdPulumiStr`?

Reply:
I used an hash to be closer to the previous version, but actually as triggers is an Array of StringInput, passing them directly should be enough